test/hotspot/jtreg/compiler/vectorapi/TestReverseByteTransforms.java

```diff
@@ -0,0 +1,306 @@
+ * @library /test/lib /
+ * @run driver compiler.vectorapi.TestReverseByteTransforms
+ */
+
+public class TestReverseByteTransforms {
+    static final VectorSpecies<Long> LSPECIES = LongVector.SPECIES_MAX;
+    static final VectorSpecies<Integer> ISPECIES = IntVector.SPECIES_MAX;
+    static final VectorSpecies<Short> SSPECIES = ShortVector.SPECIES_MAX;
+
+    static final int SIZE = 1024;
+    static final int ITERS = 50000;
+
+    static long [] lout = new long[SIZE];
+    static long [] linp = new long[SIZE];
+
+    static int [] iout = new int[SIZE];
+    static int [] iinp = new int[SIZE];
+
+    static short [] sout = new short[SIZE];
+    static short [] sinp = new short[SIZE];
+
+    static void init() {
+        Random r = new Random(1024);
+        for(int i = 0; i < SIZE; i++) {
+            linp[i] = r.nextLong();
+            iinp[i] = r.nextInt();
+            sinp[i] = (short)r.nextInt();
+        }
+    }
+
+    public static void main(String args[]) {
+        init();
+        TestFramework.runWithFlags("-XX:-TieredCompilation", "--add-modules=jdk.incubator.vector");
+        System.out.println("PASSED");
+    }
+
+    @Test
+    @IR(applyIfCPUFeature={"sve", "true"}, failOn = {"ReverseBytesV" , " > 0 "})
+    public void test_reversebytes_long_transform1(long[] lout, long[] linp) {
+        VectorMask<Long> mask = VectorMask.fromLong(LSPECIES, 3);
+        for (int i = 0; i < LSPECIES.loopBound(linp.length); i+=LSPECIES.length()) {
+            LongVector.fromArray(LSPECIES, linp, i)
+                     .lanewise(VectorOperators.REVERSE_BYTES, mask)
+                     .lanewise(VectorOperators.REVERSE_BYTES, mask)
+                     .intoArray(lout, i);
+        }
+    }
+
+    @Run(test = {"test_reversebytes_long_transform1"}, mode = RunMode.STANDALONE)
+    public void kernel_test_reversebytes_long_transform1() {
+        for (int i = 0; i < ITERS; i++) {
+            test_reversebytes_long_transform1(lout, linp);
+        }
+    }
+
+    @Test
+    @IR(applyIfCPUFeatureOr={"sve", "true", "simd", "true", "avx2", "true"}, counts = {"ReverseBytesV" , " > 0 "})
+    public void test_reversebytes_long_transform2(long[] lout, long[] linp) {
+        VectorMask<Long> mask = VectorMask.fromLong(LSPECIES, 3);
+        for (int i = 0; i < LSPECIES.loopBound(linp.length); i+=LSPECIES.length()) {
+            LongVector.fromArray(LSPECIES, linp, i)
+                     .lanewise(VectorOperators.REVERSE_BYTES)
+                     .lanewise(VectorOperators.REVERSE_BYTES, mask)
+                     .intoArray(lout, i);
+        }
+    }
+
+    @Run(test = {"test_reversebytes_long_transform2"}, mode = RunMode.STANDALONE)
+    public void kernel_test_reversebytes_long_transform2() {
+        for (int i = 0; i < ITERS; i++) {
+            test_reversebytes_long_transform2(lout, linp);
+        }
+    }
+
+    @Test
+    @IR(applyIfCPUFeatureOr={"sve", "true", "simd", "true", "avx2", "true"}, counts = {"ReverseBytesV" , " > 0 "})
```

Comment:
Cpu feature "sve" contains "simd", so I think only keep `"simd", "true"` is fine.